test/jdk/java/lang/instrument/GetObjectSizeTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2003, 2015, Oracle and/or its affiliates. All rights reserved.
```

Comment:
You can't replace Copyright lines of one company with other.

Reply:
Well, I am replacing the entire file. There is a recent precedent of the similar change [here](https://github.com/openjdk/jdk/commit/6d13c766#diff-0daf75421f8fdb55a5640742ef6f12730fe1b370cc864311c188ad99996b51fe). Either that should be reversed, or this one accepted.

Reply:
...or I can put new test into a separate file. But the existing test is quite inferior compared to the new one.

Reply:
It was mistake in 8253191 (I file bug). If you modify existing file (even if you keep only test name the same) you have to preserve original Copyright and add new Copyright line. You don't need create new file.
We have a lot of cases with 2 or more Copyright lines - it is normal:
https://github.com/openjdk/jdk/blob/master/test/hotspot/jtreg/compiler/vectorization/TestVectorsNotSavedAtSafepoint.java

Reply:
I file 8255067 to restore Copyright line in TestUnsignedByteCompare.java test file.

Reply:
I made sure the new test is in the new file.